src/js/events/states.js

```diff
@@ -1,14 +1,43 @@
-define([], function() {
-    return {
-        BUFFERING: 'buffering',
-        IDLE: 'idle',
-        COMPLETE: 'complete',
-        PAUSED: 'paused',
-        PLAYING: 'playing',
-        ERROR: 'error',
-
-        // These exist at the provider level, but are converted to BUFFERING at higher levels
-        LOADING: 'loading',
-        STALLED: 'stalled'
-    };
-});
+/**
```

Comment:
What do you think about putting all of these states in events and prefixing them with `STATE_`? These all are events and player states. I think it will make events.js more complete to include them:

```js
/**
* Triggered when the player attempts to play media, but is still loading media into the buffer or has stalled because there is not enough media in the buffer to continue.
*/
export const STATE_BUFFERING = 'buffering';
```